PULL_REQUEST_TEMPLATE

```diff
@@ -14,13 +25,13 @@
 @pklinef
 @shaundmorris
 @stustison
-#### How should this be tested?
+#### How should this be tested? (List steps with links to updated documentation)
```

Comment:
Nit, the steps should be directing them to the docs regardless if they made doc changes.